R/gitlab.R

```diff
@@ -0,0 +1,72 @@
+isGitlabURL <- function(url) {
+  is.string(url) && grepl("^http(?:s)?://(?:www|api).gitlab.(org|com)", url, perl = TRUE)
+}
+
+canUseGitlabDownloader <- function() {
+  all(packageVersionInstalled(devtools = "1.9.1", httr = "1.0.0"))
+}
+
+gitlabDownload <- function(url, destfile, ...) {
+  onError(1,{
+    gitlab_user    <- gitlab_user
```

Comment:
You can remove this line since we're just using a function defined in our own package.